docs/modules/device-lab-management/pages/ios-devices/add-ios-device.adoc

```diff
@@ -0,0 +1,318 @@
+
+== Connect the device to the host
+
+Make sure you have properly prepared the device for hosting on Kobiton.
+
+For Standard mode, connect the mobile device to the Cambrionix hub attached to the Mac mini host.
+
+For Lightning mode with iOS 16 and below, connect the mobile device to the Cambrionix hub attached to the GEM.
+
+For Lightning mode with iOS 17 and above, connect the device to one of the USB ports of the Mac mini host to establish trust pairing first, then follow the next section before connecting the device to the Cambrionix hub attached to the GEM.
+
+Check the device to see if it is charging after connecting. If it is not charging, the USB cable might be malfunctioning, or the Cambrionix hub is not connected to a power source.
+
+== Establish trust pairing between the device and the host
+
+Access the Mac mini host directly or via screen sharing.
+
+The steps to establish trust pairing vary between iOS 16 and below and iOS 17 and above.
+
+=== iOS 16 and below
+
+[NOTE]
+The steps in this section apply to both Standard and Lightning mode.
+
+Check the device screen. Tap **Trust** on the Trust this computer popup:
+
+image::device-lab-management:device-lab-management-ios-add-ios-trust-this-computer-trust.PNG[width=300,alt="Trust this computer popup, clicking Trust"]
+
+Open **Finder** in the Mac mini host, select the connected device name, and choose **Trust**.
+
+image::device-lab-management:device-lab-management-ios-add-ios-trust-iphone-trust.PNG[width=600,alt="Trust this iphone window, clicking Trust"]
+
+Unplug the device, then plug it in again. Wait until the device screen changes to the below before continuing (NOTE: there will also be an *automation running* overlay above the device screen):
+
+image::device-lab-management:device-lab-management-add-android-screen-changes-to-blue.PNG[width=300, alt="device screen changes and shows Kobiton name and logo"]
+
+=== iOS 17 and above
+
+[IMPORTANT]
+.Note for air-gapped Mac mini hosts (no Internet access)
+====
+To control the iOS devices, deviceConnect needs to mount a *Developer Disk Image* (DDI), which is a `.dmg` archive included with Xcode that contains executables and other files needed by Xcode to support debugging and testing on iOS devices.
+
+For iOS 17 and later, rather than Xcode providing a different DDI for every iOS version and device architecture, there is a generic DDI that Xcode must "personalize" for each device. The personalization process requires an Internet connection, as Xcode must use Apple's notarization servers to sign the personalized image. Without an Internet connection, Xcode can't personalize a DDI.
+
+If the Mac mini host does not have Internet connection, follow the section xref:#preload-ddi-air-gapped[preload DDI for air-gapped Mac mini] before continuing with this section.
+
+====
+
+Follow the appropriate steps based on whether you are using Standard or Lightning mode

```

Comment:
Add punctuation - either a period or : at end of this sentence.

Reply:
fixed